doc/mgr/rgw.rst

```diff
@@ -18,57 +19,110 @@ RGW Realm Operations
 
 Bootstrapping RGW realm creates a new RGW realm entity, a new zonegroup,
 and a new zone. It configures a new system user that can be used for
-multisite sync operations, and returns a corresponding token. It sets
-up new RGW instances via the orchestrator.
+multisite sync operations. Under the hood this module instructs the
+orchestrator to create and deploy the corresponding RGW daemons. The module
+supports both passing the arguments through the cmd line or as a spec file:
 
-It is also possible to create a new zone that connects to the master
-zone and synchronizes data to/from it.
+.. prompt:: bash #
+
+  rgw realm bootstrap [--realm-name] [--zonegroup-name] [--zone-name] [--port] [--placement] [--start-radosgw]
+
+The command supports providing the configuration through a spec file (`-i option`):
+
+.. prompt:: bash #
+
+  ceph rgw realm bootstrap -i myrgw.yaml
+
+Following is an example of RGW mutlisite spec file:
+
+.. code-block:: yaml
```

Comment:
In the example specs can you have one with endpoints in there, with an explanation of what their purpose is? If you start this section I can suggest some text.

Reply:
Sure, I added a new example with endpoints